canonical-websites/www.ubuntu.com
# Updated OpenStack section

## Done

- Updated OpenStack section
  - renamed /try-openstack to /developer
  - renamed /build-openstack to /install
  - removed old /install page
  - renamed /foundation-cloud-build to /consulting
  - renamed /managed-cloud to /managed
  - updated the redirects
  - updated the navigation
  - removed Juju from the OpenStack navigation and added it to the footer navigation

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [openstack](http://0.0.0.0:8001/openstack)
- See that the section reflects the above changes
